app/screens/home_screen.rb

```diff
@@ -24,6 +24,10 @@ def on_load
       Potion::Toast.makeText(find.activity, "Drink your potion.", Potion::Toast::LENGTH_SHORT).show
     end
 
+    append(Potion::Button, :dialog_button).on(:tap) do |sender|
+      PotionDialog.new(xml_layout: R::Layout::Alert_dialog, width: 700, height: 1200)
```

Comment:
Perhaps we can make this even simpler, perhaps default to R::Layout::Alert_dialog